### DISCUSSION TITLE
UI interface missing (white window)

### DISCUSSION THREAD

**Discussion by f7lixr** [KRTirtho/spotube#900] [Q&A]
Hi all,

i recenty issued a problem with Spotube. When I open it up, it won't load the interface. Some time ago I had it running perfectly fine. I just don't know what caused this issue, maybe my antivirus Bitdefender does. Reinstalling Spotube doesn't work either for me unfortunately. So I wanted to ask kindly, if anyone else is having this problem or even solved it. I will add a picture of the empty white window from Spotube.

![empty_Spotube](https://github.com/KRTirtho/spotube/assets/152042182/a82c428c-6395-4005-95be-538989b53af4)

Besides that is this project amazing on its own and I really enjoy it. Please help me to get my application fixed. Any advice is welcome and I will try out to fix it.

Many thanks upfront to all helpers,
f7lix

**Comment by KRTirtho** [comment]
Make sure any instance of Spotube isn't running. End task using "Task Manager" or close from system tray

**Reply by KRTirtho** [reply]
Then reopen the app

**Reply by f7lixr** [reply]
Hi @KRTirtho,
thanks for your fast response. Unfortunately this doesn't work for me. Even after a PC reboot the problem still occurs. I tried out launching  the app, closing it with Task Manager and reopen it with admin rights, but the interface is still missing.
I noticed that my mouse changed shape from normal arrow to pointing-finger shape on certain areas in Spotube where buttons are located. I tried out hitting the "play"- button, and even heard music! But I cannot see. The window is still white.

**Reply by KRTirtho** [reply]
Can you provide the `.spotube_logs` in your C:\Users\<user>\Documents?

**Comment by barnaba** [comment]
For me it was in a onedrive Documents directory.

Restarting doesn't help for me either.
[spotube_logs.txt](https://github.com/KRTirtho/spotube/files/14356681/spotube_logs.txt)

Seems like I'm getting rate limited, and also there were some issues with polish special characters in song names?
